### DISCUSSION TITLE
ERI TV 1 [ER]

### DISCUSSION THREAD

**Discussion by Carlinhos027** [iptv-org/community#362] [Show and tell]
### Channel ID

ERITV1.er

### Stream URL

https://player.mayya.et/play/d0019c9d-5088-47a2-bb9a-ec5fff5b9b4a/eri_1.m3u8

### Notes

Eritrean's main state-owned television network.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
https://player.mayya.et/play/d0019c9d-5088-47a2-bb9a-ec5fff5b9b4a/eri_1.m3u8 (no image)
<img width="849" alt="image" src="https://user-images.githubusercontent.com/7253922/232244593-d2e66a51-4d24-44be-9678-d9402d6076f7.png">
